docs/topics/mta-rn-known-issues-6-2-0.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-6-2-0_{context}"]
+= Known issues
+
+MTA version 6.2.0 has the following issues.
+
+.Analysis fails when fetching rules from a repository with a folder that contains spaces in its name
+
+When fetching custom rules from a repository during an analysis, if the `Root path` field contains spaces, the `mta-cli` command is not properly composed and the analysis fails. Spaces in directories specified in `userRulesDirectory` cannot be supported because it accepts a space-delimited list of directory paths. link:https://issues.redhat.com/browse/MTA-458[MTA-458]
```

Comment:
Please remove the last line, I don't think it's necessary to call out why it doesn't work.

Reply:
removed